### PR TITLE
Neurotoxin does far less direct hit stamina damage

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
Neurotoxin damage numbers turned from 40->25 for normal, and 40->30 for heavy.

## Why It's Good For The Game
Neurotoxin does too much damage for it's effects on hit, it's better that it gets lowered.
Considering you get a 15% damage bonus every tier, it'll still do quite a bit, just scale far less insanely high (up to the 60s.)

## Changelog
:cl:
balance: Light neurotoxin hits do 36%~ less damage, scaling unchanged. Heavy Neurotoxin hit (Prae) do 30. (25% less.)
/:cl:

### CODE REVIEW

File: code/modules/projectiles/ammo_datums.dm [hunk 1]
```diff
@@ -1835,7 +1835,7 @@ datum/ammo/bullet/revolver/tp44
 	spit_cost = 50
 	sound_hit = "alien_resin_build2"
 	sound_bounce = "alien_resin_build3"
-	damage = 20 //minor; this is mostly just to provide confirmation of a hit
+	damage = 10 //minor; this is mostly just to provide confirmation of a hit
```
Comment (Pariah919) [inline code/modules/projectiles/ammo_datums.dm:1838]:
```suggestion
	damage = 20 //minor; this is mostly just to provide confirmation of a hit
```

**Comment by Lefunny** [comment]
Neuro already had a big nerf with its chem purge being removed, it is also fair less used than acid also marines already got a module to just ignore it when needed

**Comment by Ragecrabbo** [comment]
Why even use neuro over acid if this gets merged? One deals fairly good instant burn damage while the other does delayed toxin damage that is often barely noticable aswell as some stam loss and can be easily purged by chems.

**Comment by Naaanii** [comment]
Too big of a nerf considering chem purge removal as well as mentioned above. Maybe nerf the cloud, but I doubt that it's that relevant or whether it even warrants a nerf.

**Comment by Pariah919** [comment]
>Why even use neuro over acid if this gets merged? One deals fairly good instant burn damage while the other does delayed toxin damage (or none at all if a marine wears mimir lol) aswell as some stam loss and can be easily purged by chems.
>
Neurotoxin damage doesn't do toxin damage unless they're stamcritted, and this doesn't even effect anything about that side of it.
>some stam loss
>
Stam loss compounds since marines use it to run, if marines have no stam then they can't run, meaning they can't give chase, making chasing either impossible or awful to do as marine. Ignoring that Neurotoxin injects on hit no matter what, has a cloud, does a direct hit. A marine without mimir is pretty cucked, even with mimir the effects are still noticeable, though the armor protection of mk2 could probably do with a reduction later.

**Comment by Lefunny** [comment]
> 

Already did in game tests with crab help and i can assert it is a lie, direct neuro hits does far close to nothing agaist marines with mimir.

**Comment by BraveMole** [comment]
I'm fine with marines not being able to run if they are constantly hit by neuro

**Pull request closed without merging**